### PR TITLE
Pull request for strickler.

### PR DESCRIPTION
**Pull request by leonmezu1**

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Hi !  
#### Wow, your project looks :+1: 

#### But need some Required Changes
Something wrong with stickler, it doesn`t work.
Please check HTML file with this validator https://validator.w3.org/#validate_by_upload now it shows me errors.

#### Page structure

##### Navigation bar

* [x] It contains Newsweek logo.
* [x] It contains current date.
* [ ] It contains Sign in and Subscribe links. You have only Subscribe link.

##### Footer

* [x] It contains Newsweek logo.
* [x] It contains social media icons.
* [ ] It contains main navigation links.

#### Implementation details

* [x] It refrences [Bootstrap CSS file](https://getbootstrap.com/docs/4.3/getting-started/introduction/#css)
* [x] It adds [vieport metatag](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag)
* [ ] It uses Bootstrap Css classes **everywhere** for styling ( for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options) ). Your CSS file shouldn`t be so big.

Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding 😄*

File: index.html [hunk 1]
```diff
@@ -1,12 +1,1117 @@
...
 <body>
-  
+	<!-- Header and nav -->
+	<header class="fixed-top" id="navi">
+		<div class="d-flex justify-content-between" id="navigation-bar">
+			<div class="p-2">
+				<date id="datenav">Thu, Oct 31, 2019</date>
+			</div>
+			<div class="p-2" id="signin">
+				<span>SIGN IN</span><a href="#" id="subs">SUBSCRIBE <i class="fas fa-chevron-right" id="chevron"></i></a>
```
Comment (Anna-Myzukina) [inline index.html]: SIGN IN should be a link

File: index.html [hunk 2]
```diff
@@ -1,12 +1,1117 @@
...
+						<li class="li-item">Advertise</li>
+						<li class="li-item">Copyright</li>
+						<li class="li-item">Terms & Conditions</li>
+						<li class="li-item">Privacy Policy</li>
+						<li class="li-item">Cookie Policy</li>
+						<li class="li-item">Terms of Sale</li>
+						<li class="li-item">Archive</li>
+						<li class="li-item">Announcements</li>
+						<li class="last li-item">Consent preferences</li>
+					</ul>
```
Comment (Anna-Myzukina) [inline index.html]: It should contain the main navigation links.

File: index.html [hunk 3]
```diff
@@ -1,12 +1,1117 @@
...
+						<ul class="list col-12 row justify-content-start justify-content-md-end text-left text-lg-right py-2 mx-4"
+							id="right">
+							<span class="editions mr-5">Editions:</span>
+							<li class="first li-item">U.S. Edition</li>
+							<li class="li-item">Middle East</li>
+							<li class="li-item">日本</li>
+							<li class="li-item">한국</li>
+							<li class="li-item">Pakistan</li>
+							<li class="li-item">Polska</li>
+							<li class="last li-item">România</li>
```
Comment (Anna-Myzukina) [inline index.html]: It should be links

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  🛑

### Hello @leonmezu1   🌟

**Good work implementing the changes, but we need to fix some more issues before moving on ⬇️** 

1. Please implement all the changes required in the previous review before submitting again.

2. Please delete the unnecessary files and folders from the repo (e.g: `.vscode` folder, and all the JavaScript files).

3. Set up Stickler to lint the CSS on your project, find out more on how to do that in [here](https://github.com/microverseinc/linters-config/tree/master/css)

4. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input)

5. Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by leonmezu1** [comment]
Hi!
Thanks for your valuable reviews!

I managed to fix the most of the issues.
The Navbar and Footers elements now have an ```<a>``` parent where it was required.

I don't really know why stickler is taking eternities to load and check, I already uninstalled and reinstalled it with no positive result.
The JavaScript files in the repository are mainly to allow stickler work in VS Code, I also  created a script.js file which contains the rules for some Navbar elements.

Regarding the CSS file size, to my defense I can only argue that the Bootstrap classes have a prefixed size for margin and padding which didn't allowed me to position the elements in the desired place another thing is I'm using a Sass preprocessor just for adding compatibility across browser, maybe it's adding more lines than the expected.

**Comment by ebukaume** [comment]
## Changes Required! :stop_sign: 

Hi @leonmezu1 :wave: 

You have really done well so far. You are almost done but we can't move forward without making sure there are no lint errors (stickler-ci).

Please refer to [this guide](https://github.com/microverseinc/linters-config/tree/master/css#troubleshooting) to help you troubleshoot why stickler is not working for this repo.

While at it, you can make your README to be more professional. Use [this template](https://github.com/othneildrew/Best-README-Template) as inspiration.

[SIDE NOTE]
Finally, as a note, you really wrote too much SCSS. You could easily achieve most of those styling with bootstrap classes.

**Please submit another code review request after fixing the above issues**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Comment by leonmezu1** [comment]
Again **thanks a lot** for your valuable reviews.

After fighting some hours with Stickler I managed to fix the whole problem it was due to the commit verification settings.
Now the Linter is working and the code is error free.

I updated the README.md file so it looks better and it's more descriptive now. 
Also, I checked the CSS sheet to shrink styling deleting more than 80 lines of unnecessary code.

Looking forward the new code review 👀

**Comment by ebukaume** [comment]
## Approved! :champagne: 

Hi @leonmezu1 :wave: 

You have really done well. I like your dedication. **You are approved to merge this branch.**


Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**